app/views/external_users/claims/disbursements/_disbursement_fields.html.haml

```diff
@@ -7,7 +7,7 @@
         %label.form-label
           = t('.disbursement_type')
         %a{id: "disbursement_#{@disbursement_count}_disbursement_type"}
-        = f.collection_select :disbursement_type_id, DisbursementType.active, :id, :name, { include_blank: ''.html_safe }, {class: 'form-control typeahead', style: 'width:350px'}
+        = f.collection_select :disbursement_type_id, DisbursementType.active, :id, :name, { include_blank: ''.html_safe }, {class: 'form-control typeahead', style: 'width:330px'}
```

Comment:
**[not a blocker]** Would it be possible to define `width:330px` in the actual stylesheets instead of the HTML directly?